PVB Linx Groep 6/Assets/Code/Scripts/UI/P_DamageNotifier.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+public class P_DamageNotifier : MonoBehaviour
+{
+    [SerializeField] private Text p_DamageText;
+    [SerializeField] private float delayBetweenNotifier_P;
```

Comment:
Rename both variables to no longer contain single letters and underscores, these are only for clarifying access level